scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/policy/edf/EDFPolicyExtended.java

```diff
@@ -65,23 +65,27 @@
 
     private static final Set<JobId> alreadyNotified = new HashSet<>();
 
-    private synchronized void logIfJobWillMissItsDeadline(List<JobDescriptor> jobs) {
+    private synchronized void logMostLikelyMissedJobs(List<JobDescriptor> jobs) {
         Date now = new Date();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "now". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "now" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)